pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -282,3 +288,15 @@ func (c *DeploymentConfigController) updateStatus(config *deployapi.DeploymentCo
 	glog.V(4).Infof("Updated the status for %q (observed generation: %d)", deployutil.LabelForDeploymentConfig(config), config.Status.ObservedGeneration)
 	return nil
 }
+
+func rcDeepCopy(rc kapi.ReplicationController) (*kapi.ReplicationController, error) {
+	objCopy, err := kapi.Scheme.DeepCopy(rc)
+	if err != nil {
+		return nil, err
+	}
+	rcCopy, ok := objCopy.(kapi.ReplicationController)
+	if !ok {
+		return nil, fmt.Errorf("expected ReplicationController, got %#v", objCopy)
```

Comment:
I've changed the object to a pointer and fyi https://github.com/kubernetes/kubernetes/issues/26591